src/Motion.js

```diff
@@ -1,9 +1,9 @@
 export function managePerimeterCollision(state, obj){
   if(atLowerBound(state.screen, obj)){
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
     obj.gravity = false;
   } else if (atUpperBound(obj)) {
-    state.currentScore += 10;
+    if(state.inGame){state.currentScore += 10}
```

Comment:
both of these are why our score kept incrementing. we won't see the levels change in the background but it was causing a problem where the score in local storage was off from the firebase score bc it was incrementing in the background.